src/utils/fetchUtils.ts

```diff
@@ -224,3 +231,62 @@ export async function fetchLanguages(): Promise<ILanguage[]> {
   });
   return languages;
 }
+
+function sortByHourAndMin(a: IEvent, b: IEvent) {
+  const aStart = a.dateStart;
+  const bStart = b.dateStart;
+  if (aStart.getHours() - bStart.getHours() === 0) {
+    return aStart.getMinutes() - bStart.getMinutes();
+  }
+  return aStart.getHours() - bStart.getHours();
+}
+
+// Since some dates in API have actual hours and others are always at 00:00
+// (meaning entire days presumably), we compare by start and end of day
+function isIncludedInDateRange(start: string, end: string) {
```

Comment:
Might be obvious from the function name, but maybe name these parameters `startDateString` and `endDateString` to be super clear.

Reply:
+ specify return type